components/write_to_file/README.md

```diff
@@ -0,0 +1,75 @@
+# Write to file
+
+<a id="write_to_file#description"></a>
+## Description
+A Fondant component to write a dataset to file on a local machine or to a cloud storage bucket. The dataset can be written as csv or parquet.
+
+<a id="write_to_file#inputs_outputs"></a>
+## Inputs / outputs 
+
+<a id="write_to_file#consumes"></a>
+### Consumes 
+
+**This component can consume additional fields**
+- <field_name>: <dataset_field_name>
+This defines a mapping to update the fields consumed by the operation as defined in the component spec.
+The keys are the names of the fields to be received by the component, while the values are 
+the name of the field to map from the input dataset
+
+See the usage example below on how to define a field name for additional fields.
+
+
+
+
+<a id="write_to_file#produces"></a>  
+### Produces 
+
+
+**This component does not produce data.**
```

Comment:
Correct bit it might confuse users.

Maybe word it like: `This component does not produce a dataframe for downstream processing.`